### PR TITLE
Prepare for releasing 1.4.8

### PR DESCRIPTION
**Pull request by arcresu**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -4,68 +4,65 @@ Changelog
...
+     according to the :ref:`preferred` configuration options.
+     Thanks to :user:`archer4499`.
+     :bug:`3017`
+   * Fingerprint values are now properly stored as strings, which prevents
+     strange repeated output when running ``beet write``.
+     Thanks to :user:`Holzhaus`.
+     :bug:`3097` :bug:`2942`
 * :doc:`/plugins/convert`: The plugin now has a ``id3v23`` option that allows
-  to override the global ``id3v23`` option.
+  youto override the global ``id3v23`` option.
```
Comment (jackwilsdon) [inline docs/changelog.rst]:
```suggestion
  you to override the global ``id3v23`` option.
```

**Comment by arcresu** [comment]
I split the changelog into several themed sections, but sometimes the boundary between "fix" and "feature" is not so clear. Maybe it's better to just group by plugin for the most part?

File: docs/changelog.rst [hunk 2]
```diff
@@ -4,68 +4,65 @@ Changelog
...
+  for the :ref:`aunique` template function.
+* The ``albumdisambig`` field no longer includes the MusicBrainz release group
+  disambiguation comment. A new ``releasegroupdisambig`` field has been added.
+  :bug:`3024`
+* The :ref:`modify-cmd` command now allows resetting fixed attributes. For
+  example, ``beet modify -a artist:beatles artpath!`` resets ``artpath``
+  attribute from matching albums back to the default value.
+  :bug:`2497`
+* A new importer option, :ref:`ignore_data_tracks`, lets you skip audio tracks
+  contained in data files :bug:`3021`
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
  contained in data files. :bug:`3021`
```

File: docs/changelog.rst [hunk 3]
```diff
@@ -74,58 +71,142 @@ New features:
...
+     :bug:`3205` :bug:`800`
+   * MPD protocol command ``decoders`` is now supported.
+     :bug:`3222`
+   * The plugin now uses the main beets logging system.
+     The special-purpose ``--debug`` flag has been removed.
+     Thanks to :user:`arcresu`.
+     :bug:`3196`
+* :doc:`/plugins/mbsync`: The plugin no longer queries MusicBrainz when either
+  the ``mb_albumid`` or ``mb_trackid`` field is invalid.
+  See also the discussion on Google Groups_
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
  See also the discussion on `Google Groups`_
```

File: docs/changelog.rst [hunk 4]
```diff
@@ -74,58 +71,142 @@ New features:
...
+  Thanks to :user:`arogl`.
+* :doc:`/plugins/export`: The plugin now also exports ``path`` field if user
+  explicitly specifies it with ``-i`` parameter. This only works when exporting
+  library fields.
+  :bug:`3084`
+
+.. _Groups: https://groups.google.com/forum/#!searchin/beets-users/mbsync|sort:date/beets-users/iwCF6bNdh9A/i1xl4Gx8BQAJ
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
.. _Google Groups: https://groups.google.com/forum/#!searchin/beets-users/mbsync|sort:date/beets-users/iwCF6bNdh9A/i1xl4Gx8BQAJ
```

File: docs/changelog.rst [hunk 5]
```diff
@@ -74,58 +71,142 @@ New features:
...
+     :bug:`3222`
+   * The plugin now uses the main beets logging system.
+     The special-purpose ``--debug`` flag has been removed.
+     Thanks to :user:`arcresu`.
+     :bug:`3196`
+* :doc:`/plugins/mbsync`: The plugin no longer queries MusicBrainz when either
+  the ``mb_albumid`` or ``mb_trackid`` field is invalid.
+  See also the discussion on Google Groups_
+  Thanks to :user:`arogl`.
+* :doc:`/plugins/export`: The plugin now also exports ``path`` field if user
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
* :doc:`/plugins/export`: The plugin now also exports ``path`` field if the user
```

File: docs/changelog.rst [hunk 6]
```diff
@@ -74,58 +71,142 @@ New features:
...
+  :bug:`2951`
+* Added whitespace padding to missing tracks dialog to improve readability.
+  Thanks to :user:`jams2`.
+  :bug:`2962`
+* The :ref:`move-cmd` command now lists the number of items already in-place.
+  Thanks to :user:`RollingStar`.
+  :bug:`3117`
+* Modify selection can now be applied early without selecting every item.
+  :bug:`3083`
+* Improve error reporting during startup if sqlite returns an error. The
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
* Beets now emits more useful messages during startup if SQLite returns an error. The
```

File: docs/changelog.rst [hunk 7]
```diff
@@ -74,58 +71,142 @@ New features:
...
+  sqlite error message is now attached to the beets message.
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
  SQLite error message is now attached to the beets message.
```

File: docs/changelog.rst [hunk 8]
```diff
@@ -74,58 +71,142 @@ New features:
...
+  :bug:`3007` :bug:`3215`
+
+There are many fixes related to compatibility with our dependencies including
+addressing changes interfaces:
 
 * On Python 2, pin the Jellyfish requirement to version 0.6.0 for
   compatibility.
-* A new importer option, :ref:`ignore_data_tracks`, lets you skip audio tracks
-  contained in data files :bug:`3021`
+* Fix compatibility Python 3.7 and its change to a name in the ``re`` module.
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
* Fix compatibility with Python 3.7 and its change to a name in the ``re`` module.
```

File: docs/changelog.rst [hunk 9]
```diff
@@ -74,58 +71,142 @@ New features:
...
+* :doc:`/plugins/web`: Fix an error when using more recent versions of Flask
+  with CORS enabled.
+  Thanks to :user:`rveachkc`.
+  :bug:`2979`: :bug:`2980`
+* Avoid some deprecation warnings with certain versions of the MusicBrainz
+  library.
+  Thanks to :user:`zhelezov`.
+  :bug:`2826` :bug:`3092`
 * Restore iTunes Store album art source, and remove the dependency on
   python-itunes_, which had gone unmaintained and was not py3 compatible.
```
Comment (sampsyo) [inline docs/changelog.rst]:
```suggestion
  python-itunes_, which had gone unmaintained and was not Python 3-compatible.
```

File: docs/changelog.rst [hunk 10]
```diff
@@ -136,86 +217,30 @@ Fixes:
...
-* :doc:`/plugins/fetchart`: Added network connection error handling to backends
-  so that beets won't crash if a request fails.
-  Thanks to :user:`Holzhaus`.
-  :bug:`1579`
-* Fetchart now respects the ``ignore`` and ``ignore_hidden`` settings. :bug:`1632`
-* :doc:`/plugins/badfiles`: Avoid a crash when the underlying tool emits
-  undecodable output.
-  :bug:`3165`
+* :doc:`/plugins/lastgenre`: Allow to set the configuration option
+  ``prefer_specific`` without setting ``canonical``.
```
Comment (sampsyo) [inline docs/changelog.rst]: Perhaps some simplified wording:

> It's now possible to set the `prefer_specific` option without also setting `canonical`.

File: docs/changelog.rst [hunk 11]
```diff
@@ -225,6 +250,12 @@ For developers:
   ``playlist:name`` although there is no field named ``playlist``.
   See :ref:`extend-query` for details.
 
+For packagers:
+
+* Note the changes to the dependencies on jellyfish and munkres.
+* The optional python-itunes dependency has been removed.
+* Python versions 3.7 and 3.8 are now supported.
```
Comment (sampsyo) [inline docs/changelog.rst]: Perhaps use ` (ReST's "interpreted text") to wrap the package names.

**Comment by arcresu** [comment]
I went through the plugin docs and updated the optional dependencies in setup.py which had fallen out of date. Along the way I found some cases that looked like markup issues where code markup was intended but italics had been used instead. I also saw that there were two plugins called "Smart Playlist Plugin" but it looked like one of them had accidentally included the "smart" in its title.

I checked the release checklist and it seems that all that's missing is a short summary of the release at the top of the changelog, and of course actually making the release! I'm not sure about the summary since there doesn't seem to have been a specific release focus this time around. Maybe I can leave that to you @sampsyo?

**Pull request merged**